### ISSUE TITLE
R2S，6.6.63无法启动

### ISSUE THREAD

**Issue by strifescl** [kiddin9/Kwrt#1953]
R2S 6.6.63无法启动，还望大佬修复

**Comment by Sia7ash** [comment]
> R2S 6.6.63 cannot start, hope you can fix it

same here

**Comment by Thessalhydra** [comment]
12月3日的依然无法启动,估计是上游的问题吧？

**Comment by Sia7ash** [comment]
> The one on December 3rd still cannot be started. I guess it’s an upstream problem?

yeah , same here , I try it everyday with new compile but not working ! I do not know how many days i have to try to make it work again!

**Comment by jpsn123** [comment]
r3s也出现了  debug发现是上游uboot问题

**Comment by Sia7ash** [comment]
> R3s also had debugging and found that it was an upstream uboot problem

did you fix it?

**Comment by Thessalhydra** [comment]
> > R3s also had debugging and found that it was an upstream uboot problem
> 
> did you fix it?

上游回退了uboot https://github.com/coolsnowwolf/lede/commit/062dc200788c30fc85e7e4ffe005655ea8878f8d
不过12月16日的还是无法启动，等下次更新再试试吧

**Comment by jpsn123** [comment]
已提交PR, 等LEDE大神合并  https://github.com/coolsnowwolf/lede/pull/13232

**Comment by wdlid** [comment]
Still no fix.

**Comment by Thessalhydra** [comment]
修复了吗？
